os/contiki-main.c

```diff
@@ -86,9 +86,17 @@ main(void)
   platform_init_stage_two();
 
   LOG_INFO("Starting " CONTIKI_VERSION_STRING "\n");
-
-  LOG_INFO(" Net: %s\n", NETSTACK_NETWORK.name);
-  LOG_INFO(" MAC: %s\n", NETSTACK_MAC.name);
+#define TSCH_HOPPING_SEQUENCE_16_16 (uint8_t[]){ 16, 17, 23, 18, 26, 15, 25, 22, 19, 11, 12, 13, 24, 14, 20, 21 }
```

Comment:
I am not hugely keen on defines intermingled with code.

Reply:
oops this was leftover from a test, never intended for commit!